Adding third-party integration info (#6569)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/double-key-encryption.md

If you want, once you're finished setting up DKE, you can migrate content that you've protected using HYOK labels to DKE labels. To migrate, you'll use the AIP scanner. To get started using the scanner, see [What is the Azure Information Protection unified labeling scanner?](/azure/information-protection/deploy-aip-scanner).

If you don't migrate content, your HYOK protected content will remain unaffected.

> ## Other Deployment Options

Reviewer: ```suggestion
## Other deployment options
```

> We realize that for some customers in highly regulated industries, this standard reference implementation using software based keys may not be sufficient to meet their enhanced compliance obligations and needs.

Reviewer: ```suggestion
We realize that for some customers in highly regulated industries, this standard reference implementation using software-based keys may not be sufficient to meet their enhanced compliance obligations and needs.
```

> We have partnered with various third-party Hardware Security Module (HSM) vendors to bring support for enhanced key management options to the Double Key Encryption Service:

Reviewer: ```suggestion
We have partnered with various third-party hardware security module (HSM) vendors to bring support for enhanced key management options to the DKE service, including:
```

 - [Entrust](https://www.entrust.com/digital-security/hsm/services/packaged-services/double-key-encryption-integration#:~:text=Entrust%20Double%20Key%20Encryption%20for%20Microsoft%20AIP%2C%20offered,trust%20for%20the%20protection%20of%20sensitive%20cryptographic%20keys.) 

- [Thales](https://cpl.thalesgroup.com/cloud-security/encryption/double-key-encryption) 

> Please reach out directly to these vendors for more information and guidance on their in-market DKE HSM solutions. 

Reviewer: ```suggestion
Reach out directly to these vendors for more information and guidance on their in-market DKE HSM solutions. 
```
